File: SproInstall/SocketPro/tutorial/CSharp/SampleThree/Server/TOne_i.cs
public static class TOneConst
{
	//defines for service CTOne
	public const int sidCTOne = ((int)USOCKETLib.tagOtherDefine.odUserServiceIDMin + 20);

	public const short idQueryCountCTOne = ((short)USOCKETLib.tagOtherDefine.odUserRequestIDMin + 0);
	public const short idQueryGlobalCountCTOne = (idQueryCountCTOne + 1);
	public const short idQueryGlobalFastCountCTOne = (idQueryGlobalCountCTOne + 1);
	public const short idSleepCTOne = (idQueryGlobalFastCountCTOne + 1);
	public const short idEchoCTOne = (idSleepCTOne + 1);
}

File: SproInstall/SocketPro/tutorial/VBNet/SampleThree/Server/TOne_i.vb
Public Class TOneConst
	'defines for service CTOne
    Public Const sidCTOne As Integer = (CInt(USOCKETLib.tagOtherDefine.odUserServiceIDMin) + 20)
    Public Const idQueryCountCTOne As Short = (CShort(USOCKETLib.tagOtherDefine.odUserRequestIDMin) + 0)
	Public Const idQueryGlobalCountCTOne As Short = (idQueryCountCTOne + 1)
	Public Const idQueryGlobalFastCountCTOne As Short = (idQueryGlobalCountCTOne + 1)
	Public Const idSleepCTOne As Short = (idQueryGlobalFastCountCTOne + 1)
	Public Const idEchoCTOne As Short = (idSleepCTOne + 1)
End Class